lib/os/fdtable.c

```diff
@@ -224,21 +224,37 @@ int z_alloc_fd(void *obj, const struct fd_op_vtable *vtable)
 
```

Comment:
Typo in commit message of `lib/os: fdtable: add locking to posix api`
s/sokcets/sockets/

Reply:
Fixed